src/main/resources/oracle/populate.sql

```diff
@@ -0,0 +1,37 @@
+-- All these insert/update/delete will be captured.
```

Comment:
This isn't an issue with your PR specifically, but we should move the `populate.sql` files to `src/test`.

Reply:
Follow up, tracked by https://github.com/gingersnap-project/db-syncer/issues/71